src/Features/Core/Portable/GenerateConstructorFromMembers/AbstractGenerateConstructorFromMembersCodeRefactoringProvider.ConstructorDelegatingCodeAction.cs

```diff
@@ -92,7 +92,7 @@ protected override async Task<Document> GetChangedDocumentAsync(CancellationToke
                     _state.ContainingType,
                     CodeGenerationSymbolFactory.CreateConstructorSymbol(
                         attributes: default,
-                        accessibility: Accessibility.Public,
+                        accessibility: _state.ContainingType.IsAbstractClass() ? Accessibility.Protected : Accessibility.Public,
```

Comment:
note: this only makes the new constructor *less* accessible than before.  So this is a safe change to make